2015-01-02 12:44:07.165 [error] <0.364.0> gen_server sctp_listener terminated with reason: no such process or port in call to gen_server:call(sctp_client_sup, {start_child,[]}, infinity)
2015-01-02 16:36:19.295 [error] <0.364.0> CRASH REPORT Process <0.364.0> with 0 neighbours exited with reason: eaddrinuse in gen_server:init_it/6 line 322
